tari-project/universe
# fix: disable tor bridges settings for macos

Description
---

- update Tor settings:
  - check for OS
  - disable extra Tor settings UI if macOS, with explainer copy
- other changes:
  - fix text button colour check
  - rm console.gebug

How Has This Been Tested?
---


- locally: 

https://github.com/user-attachments/assets/bd8f598e-ebf5-4bda-bed4-0214dabadcfc


What process can a PR reviewer use to test or verify this change?
---

- run on a mac 
- try use Tor settings
- only off/on should be allowed on mac
